resources/references/adr/performance/2022-03-25-profiler-integrations.md

```diff
@@ -107,7 +107,7 @@ class CartCalculator
 ```
 
 These spans are then displayed in the timeline of the corresponding profilers:
-![](../../../../.gitbook/assets/adr/tideways_benchmark.PNG "Tideways benchmark")
+![](../../../../.gitbook/assets/adr/tideways_benchmark.png "Tideways benchmark")
```

Comment:
this should be lowercased automatically  in https://github.com/shopware/docs/blob/master/.github/scripts/update-adrs.sh

Reply:
With previous comment in mind, since we'll need to create a PR in `shopware/platform`, I can as well fix this image there and then update ADRs.

Reply:
Good Idea. I hadn't realised the source adrs were faulty.